sound/soc/sdca/sdca_functions.c

```diff
@@ -165,6 +165,10 @@ void sdca_lookup_functions(struct sdw_slave *slave)
 	struct device *dev = &slave->dev;
 	struct acpi_device *adev = to_acpi_device_node(dev->fwnode);
 
+	if (!adev) {
+		dev_dbg(dev, "Not an ACPI device\n");
```

Comment:
I would move this to info level to be visible and rephrase the print to something like 'No matching ACPI device found, ignoring peripheral`

Reply:
Updated

Reply:
Btw, do you need/want Fixes tag?

Reply:
> Btw, do you need/want Fixes tag?

Yes, done.